### PR TITLE
add rule for new issue without a description

### PR DESCRIPTION
**Pull request by SimonLab**
see #76 

This PR will check that when a user create an issue a description is also provided.

### CODE REVIEW

File: web/controllers/rules/issue/time_estimation.ex [hunk 1]
```diff
@@ -11,7 +11,7 @@ defmodule Dwylbot.Rules.TimeEstimation do
     in_progress = Enum.any?(labels, fn(l) -> l["name"] == "in-progress" end)
     estimation = labels
     |> Enum.map(fn(l) -> Regex.match?(~r/T\d{1,3}[mhd]/, l["name"]) end)
-    |> Enum.reduce(fn(x, acc) -> x || acc end)
+    |> Enum.reduce(false, fn(x, acc) -> x || acc end)
```
Comment (SimonLab) [inline web/controllers/rules/issue/time_estimation.ex:14]: I initialise the acc for the reduce function to ```false```. This avoid having an error if the list is empty.

File: test/controllers/event_controller_test.exs [hunk 2]
```diff
@@ -17,4 +17,21 @@ defmodule Dwylbot.EventTestController do
...
+    assert json_response(conn, 200)
+  end
+
+  test "POST /event/new - unassigned with inprogress", %{conn: conn} do
+    payload = "./test/fixtures/unassigned_inprogress.json"
+    |> File.read!()
+    |> PP.parse!()
+    conn = post conn, "/event/new", payload
+    assert json_response(conn, 200)
+  end
```
Comment (samhstn) [inline test/controllers/event_controller_test.exs]: At this point I would abstract out these 4 tests into 1 test:

```elixir
@fixtures ~w(add_label inprogress no_description unassigned_inprogress) |> Enum.map(&("./test/fixtures#{&1}.json"))

test "POST /event/new", %{conn: conn} do
  for fixture <- @fixtures do
    payload = fixture |> File.read! |> PP.parse
    conn = post conn, "/event/new", payload
    assert json_response(conn, 200)
  end
end
```

Or something like that

Also I may look at checking more than just `json_response(conn, 200)`, are there more specific things you can get from the response

**Review by samhstn — APPROVED** [review]
Looks good 👍

File: test/controllers/event_controller_test.exs (hunk 2 above)
Comment (SimonLab) [reply test/controllers/event_controller_test.exs]: nice, I'm going to add more unit test too which might replace those tests

**Review by nelsonic — APPROVED** [review]
@SimonLab superb work (_again!_) 👍 
(_P.S. Can we adapt this for **PR**s without a description...?_)

**Comment by SimonLab** [comment]
@nelsonic thanks, yes I will create an issue for adding the same rule on PRs :+1: 
It might just need an update of the github app configuration (select the event that trigger a webhook)

**Pull request merged**